<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZyElRunes</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="topHorizontalLayout">
       <item>
        <widget class="QListWidget" name="runesWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="2" column="1">
          <widget class="QSpinBox" name="levelBox">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="itemBox"/>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="soketBox">
           <property name="maxCount">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="charBox"/>
         </item>
         <item row="9" column="1">
          <widget class="QPushButton" name="findButton">
           <property name="text">
            <string>&amp;Find</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="levelLabel">
           <property name="text">
            <string>L&amp;evel:</string>
           </property>
           <property name="buddy">
            <cstring>levelBox</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="item">
           <property name="text">
            <string>I&amp;tem:</string>
           </property>
           <property name="buddy">
            <cstring>itemBox</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="soketLabel">
           <property name="text">
            <string>&amp;Sokets:</string>
           </property>
           <property name="buddy">
            <cstring>soketBox</cstring>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="charLabel">
           <property name="text">
            <string>&amp;Character:</string>
           </property>
           <property name="buddy">
            <cstring>charBox</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="bottomHorizontalLayout">
       <item>
        <widget class="QTableView" name="runewordsTableView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>331</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>runesWidget</tabstop>
  <tabstop>levelBox</tabstop>
  <tabstop>itemBox</tabstop>
  <tabstop>soketBox</tabstop>
  <tabstop>charBox</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>runewordsTableView</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
